tests/testthat/test-parse-variables-sheet.R

```diff
@@ -0,0 +1,157 @@
+# Test 6a: Successful parsing with all non-derived variables
+test_that("parse_variables_sheet succeeds with all non-derived variables", {
+  vars_sheet <- data.frame(
+    variable = c("age", "sex", "height"),
+    variableStart = c("[age]", "cchs::sex", "[height], pbc::height")
+  )
+
+  result <- parse_variables_sheet(vars_sheet)
+
+  expect_s3_class(result, "variables_sheet")
+  expect_equal(result$variable, vars_sheet$variable)
```

Comment:
Rather than using two assertions, can you create a variable called `expected_result` that has the expected return of the function call and compare it with the actual result using the `expect_equal` function. For example,

```
expected_result <- data.frame(vars_sheet)
class(expected_result) <- c("variables_sheet", "data.frame")
```

Also, do this for all the tests